bookstore/src/main/webapp/static/css/style.css

```diff
@@ -0,0 +1,281 @@
+.row {
+	padding: 0px;
+	margin: 0px;
+}
+
+.header {
+	background-color: #C0C0C0;
+	height: 150px;
+	border-bottom: 1px solid #e5e5e5;
+}
+
+.header .contact-information {
+	background-color: white;
+	height: 50px;
+	border: 1px solid #dddddd;
+}
+
+.header .contact-information .mail-contact {
+	font-size: 15px;
+	color: #252525;
+	float: right;
+	display: inline-block;
+}
+
+.header .contact-information .phone-contact {
+	float: left;
+	font-size: 14px;
+	color: #252525;
+	display: inline-block;
+}
+
+.header .contact-information .social-network {
+	font-size: 16px;
+	color: #252525;
+	float: right;
+	margin-right: 20px;
+}
+
+.header .contact-information .lang{
+	font-size: 15px;
+	color: #252525;
+	background-color: white;
+	padding-top: 10px;
+}
+
+.header .contact-information .login-top {
+	font-size: 15px;
+	color: #252525;
+	float: right;
+}
+.social-network ul li {
+  display:inline;
+  margin-right: 10px;
+}
+
+.header .border-right {
+	border-right: 1px solid #dddddd;
+	padding-top: 12px;
+}
+
+.header .product-search {
+	background-color: #FFFFFF;
+	height: 115px;
+}
+
+.header .product-search .glyphicon-shopping-cart, .header .product-search .heart
+	{
+	color: #dddddd;
+	padding-top: 18px;
+	font-size: 30px;
+}
+
+.header .product-search img {
+	margin-top: 7.5px;
+	height: 90px;
+}
+
+.product-search .search-content {
+	height: 50px;
+	margin-top: 30px;
+	border: 1px solid #dddddd;
+	padding: 0px;
+}
+
+.search-content input {
+	height: 94.30%;
```

Comment:
@ronaldolima99 
What is the parent class of .search-content
Should group the classes of .search-content together